src/main/java/com/deavensoft/paketomat/user/UserServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.deavensoft.paketomat.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private User user;
+    @Autowired
+    public UserServiceImpl(@Qualifier("user") User user){
+        this.user = user;
+    }
+    public List<com.deavensoft.paketomat.center.model.User> getAllUsers(){
```

Comment:
When you rename User repository to UserRepository, import User model at the top of the class.

- [ ] Import class using _import_ statement